### PR TITLE
dvc v2.10.2

### PR DESCRIPTION
**Pull request by dtrifiro**
Based off https://github.com/conda-forge/dvc-feedstock/pull/259, updating dependencies in `meta.yaml`

- updated v2.10.2
- update meta.yaml

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -83,8 +83,9 @@ outputs:
         - fsspec >=2021.10.1
         - aiohttp-retry >=2.4.5
         - diskcache >=5.2.1
-        - scmrepo ==0.0.16
-        - dvc-render ==0.0.4
+        - scmrepo ==0.0.19
+        - dvc-render ==0.0.5
+        - dvclive[image] >=0.7.3
```
Comment (efiop) [inline recipe/meta.yaml:88]:
```suggestion
        - dvclive >=0.7.3
```
the `[image]` is a requirements for dvc tests, and also not available in dvclive, if I understand correctly.

Comment (efiop) [reply recipe/meta.yaml:88]: cc @daavoo

Comment (daavoo) [reply recipe/meta.yaml]: correct

**Comment by dtrifiro** [comment]
It seems that CI failed because of a failure to pull on azure plipelines (Git error 128, a fairly common occurrence), but 2.10.2 has been deployed to conda: https://anaconda.org/conda-forge/dvc

**Pull request merged**